src/python/CRABClient/Commands/status.py

```diff
@@ -171,6 +171,10 @@ def makeStatusReturnDict(self, crabDBInfo, combinedStatus, dagStatus = '',
             this dictionary in their scripts.
         """
 
+        if shortResult is None: shortResult = {}
```

Comment:
I really can't understand why we should replace
```
a={}
```
with
```
a = None
if a is None: a = {}
```
are you really sure that we need this commit ?

Reply:
It removes the first warning from pylint in the previous Jenkins results:
https://cmssdt.cern.ch/dmwm-jenkins/view/All/job/DMWM-CRABClient-PR-test/399/artifact/artifacts/PullRequestReport.html